tests/matrix.cpp

```diff
@@ -76,6 +76,17 @@ BOOST_AUTO_TEST_CASE( lookat )
     BOOST_CHECK_EQUAL( up, newUp );
 }
 
+BOOST_AUTO_TEST_CASE( nonInvertible )
+{
+    vmml::Matrix2f matrix;
+    matrix( 0, 0 ) = 0.f;
+    matrix( 1, 1 ) = 0.f;
+    matrix.inverse();
+    // A NaN never compares equal to itself.
```

Comment:
This test is testing nothing? use http://en.cppreference.com/w/cpp/numeric/math/isnan ?